README.md

```diff
@@ -42,3 +42,12 @@ elm-analyse -p 3001 -s   # show extended analyse in the browser on `localhost:30
 yarn install
 yarn build
 ```
+
+## Configurations
+
+Our app have a somewhat complex configuration stack, allowing the app to run without anychanges, connecting to the staging environmnent. Here is an outline of all the configuration files and how the interact:
+
+- `src/scripts/config.js` is the bottom level config. All the defaults are stored here for `DEV`, `PROD`, and `LOCAL` environments.
+- `env-config.js` is the config for the dev/prod/demo servers. This file overwrites the defaults from the `config.js` above. Currently, this file _is not used_ in the local dev environment, on the localhost it won't be loaded correctly. In the repo, this file contains the data for the Staging server (in Demo it will be different, in Muda, it will be different, etc.).
```

Comment:
Since newcomers are hardly familiar with what Muda is, I think the sentence in parentheses could be like this: "_(in Demo server it will be different, in SomeCommunity server it will be different, etc.)_".